Refactoring the Watchdog alert page (#20853)
DataDog/documentation: content/en/watchdog/_index.md

    algolia:
        rank: 70
---
## Overview

> Watchdog is Datadog’s AI engine, providing you with automated alerts, insights, and root cause analyses that draw from observability data across the entire Datadog platform. Watchdog continuously monitors your infrastructure and calls attention to the signals that matter most, helping you quickly detect, troubleshoot, and resolve issues. 

Reviewer: ```suggestion
Watchdog is Datadog's AI engine, providing you with automated alerts, insights, and root cause analyses that draw from observability data across the entire Datadog platform. Watchdog continuously monitors your infrastructure and calls attention to the signals that matter most, helping you to detect, troubleshoot, and resolve issues. 
```

All Watchdog features come built-in, and do not require setup.

> {{< youtube url="https://www.youtube.com/watch?v=UMNXdrI0J8A" >}}

Reviewer: ```suggestion
{{< youtube UMNXdrI0J8A >}}
```

### Proactive alerts

Watchdog proactively computes a baseline of expected behavior for your systems, applications, and deployments. This baseline is then used to detect anomalous behavior.

{{< whatsnext desc="The following pages provide more information:">}}
  {{< nextlink href="/watchdog/alerts">}}<u>Watchdog Alerts</u>: Learn how to view and interpret Watchdog Alerts: the information provided by each alert, what alerts cover, and where to find Watchdog alerts throughout Datadog.{{< /nextlink >}}
  {{< nextlink href="/watchdog/faulty_deployment_detection">}}<u>Faulty Deployment Detection</u>: Learn about Automatic Faulty Deployment Detection, a Watchdog capability that finds faulty code deployments.{{< /nextlink >}}
{{< /whatsnext >}}

To customize Watchdog algorithms:
  * [Anomaly Algorithm][7]
  * [Forecast Algorithm][8]
  * [Outlier Algorithm][9]

### Investigation assistance

To help investigation, Watchdog shows context-based insights in all explorers, searches for root causes, and determines user impact.

{{< whatsnext desc="The following pages provide more information">}}

>   {{< nextlink href="/watchdog/insights">}}<u>Watchdog Insights</u>: Learn about Watchdog Insights, a recommendation engine that helps users quickly identify and resolve issues.{{< /nextlink >}}

Reviewer: ```suggestion
  {{< nextlink href="/watchdog/insights">}}<u>Watchdog Insights</u>: Learn about Watchdog Insights, a recommendation engine that helps users identify and resolve issues.{{< /nextlink >}}
```
